### PR TITLE
Support suppressed lint message

### PR DESCRIPTION
**Pull request by Affankhushnod1994**
- Make Codecoach aware of `SuppressedLintMessage`
- Add suppressed ESLint message to message list with severity === 0 (ignore) so codecoach doesn't comment on the MR
- Aforementioned message needs to be queried from superset.

### CODE REVIEW

File: src/Parser/ESLintParser.ts [hunk 1]
```diff
@@ -13,6 +13,18 @@ export class ESLintParser extends Parser {
       const logs = JSON.parse(content) as ESLintLog[];
       return logs
         .filter((log) => log.messages.length !== 0)
+        .concat(
```
Comment (Affankhushnod1994) [inline src/Parser/ESLintParser.ts:16]: :rotating_light: Parsing error: ')' expected.

File: src/Parser/ESLintParser.ts [hunk 2]
```diff
@@ -13,6 +13,18 @@ export class ESLintParser extends Parser {
...
+          logs
```
Comment (Affankhushnod1994) [inline src/Parser/ESLintParser.ts:17]: :warning: 'content' is defined but never used. (rule: @typescript-eslint/no-unused-vars)

**Comment by Affankhushnod1994** [comment]
## CodeCoach reports 2 issues
:rotating_light: 1 error
:warning: 1 warning

File: src/Parser/ESLintParser.ts (hunk 2 above)
Comment (Affankhushnod1994) [inline src/Parser/ESLintParser.ts:17]: :warning: 'content' is defined but never used. (rule: @typescript-eslint/no-unused-vars)

**Comment by Affankhushnod1994** [comment]
## CodeCoach reports 2 issues
:rotating_light: 1 error
:warning: 1 warning

File: src/Parser/ESLintParser.ts (hunk 2 above)
Comment (Affankhushnod1994) [inline src/Parser/ESLintParser.ts:17]: :warning: 'content' is defined but never used. (rule: @typescript-eslint/no-unused-vars)

File: src/Parser/ESLintParser.ts (hunk 1 above)
Comment (Affankhushnod1994) [inline src/Parser/ESLintParser.ts:16]: :rotating_light: Parsing error: ')' expected.

**Comment by Affankhushnod1994** [comment]
## CodeCoach reports 2 issues
:rotating_light: 1 error
:warning: 1 warning

File: src/Parser/ESLintParser.ts (hunk 2 above)
Comment (Affankhushnod1994) [inline src/Parser/ESLintParser.ts:17]: :warning: 'content' is defined but never used. (rule: @typescript-eslint/no-unused-vars)

File: src/Parser/ESLintParser.ts (hunk 1 above)
Comment (Affankhushnod1994) [inline src/Parser/ESLintParser.ts:16]: :rotating_light: Parsing error: ')' expected.

**Comment by Affankhushnod1994** [comment]
## CodeCoach reports 2 issues
:rotating_light: 1 error
:warning: 1 warning

**Review by encX — APPROVED** [review]
looks good!

**Review by encX — APPROVED** [review]

**Pull request merged**